src/global/Constants.h

```diff
@@ -51,12 +51,16 @@ static const int DEFAULT_NOF_VALUE_MANTISSA_DIGITS = 30;
 static const int DEFAULT_NOF_DATE_YEAR_DIGITS = 19;
 
 static const std::string MMAP_FILE_SUFFIX = ".meta-mmap";
-static const std::string CONFIGURATION_FILE = ".configuration";
+static const std::string CONFIGURATION_FILE = ".conf";
 
-static constexpr size_t MIN_COMPRESSION_PREFIX = 128;
-static constexpr size_t NUM_COMPRESSION_PREFIXES = 127;
+// Constants for the range of valid compression prefixes
+// all printable characters are left out
```

Comment:
Say ASCII not printable, we are assuming UTF-8 in most of QLever. Also, is this really relevant now that we always use the first byte to indicate compression. We could just as easily use `'N'` and `'C'` for (non-) compressed.